src/libraries/pretest.proj

```diff
@@ -38,9 +38,9 @@
 
   <Target Name="GetSharedFrameworkRuntimeFiles">
     <ItemGroup>
-      <ManualRuntimePackNativeFile Include="System.Security.Cryptography.Native.OpenSsl.so" />
+      <ManualRuntimePackNativeFile Include="System.Security.Cryptography.Native.Android.so" />
```

Comment:
the .Android.so should be added in addition instead of replacing, since .OpenSsl.so still exists for e.g. desktop Linux.